soc/ite/ec/it8xxx2/Kconfig

```diff
@@ -183,6 +183,17 @@ config SOC_IT8XXX2_EXCEPTIONS_IN_RAM
 	  Flash. This can significantly improve performance when under I-cache
 	  pressure.
 
+config SOC_IT8XXX2_LIBRARY_TO_RAM
+	bool "Move library to RAM"
```

Comment:
This appears to be a helper symbol only.  [Helper symbols ](https://docs.zephyrproject.org/latest/build/kconfig/tips.html#using-select-for-helper-symbols) should not have prompt text, but should provide help text.

Reply:
Done.